### ISSUE TITLE
Deleting a file does not update the database automatically

### ISSUE THREAD

**Issue by rieje** [cboxdoerfer/fsearch#64]
Not sure if this is a current limitation, but deleting a file does not update the database automatically. At first, I thought "Delete" and "Move to Trash" weren't working, then I realized the database was not updated to reflect the change.

**Comment by cboxdoerfer** [comment]
You are right, that's a current limitation of the database. This will be solved once I'll add the option to monitor the filesystem for changes.

And yes, currently renaming files isn't possible. However that's one item on my (long) todo list.

**Comment by udiNur** [comment]
What the gnome-search-tool does to solve the problem is when you get the search results, is checked whether the files exist in the file system, if one of them does not exist, it removes the file from the DB and does not display it in the results. you not need to monitor the filesystem for that..

**Comment by cboxdoerfer** [comment]
This has now been implemented as part of file system monitoring and will be part of the 0.3 release.
